pymc3/distributions/continuous.py

```diff
@@ -336,7 +336,19 @@ class HalfNormal(PositiveContinuous):
 
        f(x \mid \tau) =
            \sqrt{\frac{2\tau}{\pi}}
-           \exp\left\{ {\frac{-x^2 \tau}{2}}\right\}
+           \exp\left(\frac{-x^2 \tau}{2}\right)
+
+       f(x \mid \sigma) =\sigma
+           \sqrt{\frac{2}{\pi}}
+           \exp\left(\frac{-x^2}{2\sigma^2}\right)
+
+    .. note::
+
+       The parameters σ/τ refer to the standard deviation/precision of the
```

Comment:
these are the characters causing the encoding problems.  I would probably change then to `sd/tau`, since those are the names of the parameters in the function (and then we avoid encoding source code)